tests/core/full_node/config.py

```diff
@@ -1,5 +1,5 @@
 # flake8: noqa: E501
-job_timeout = 60
+job_timeout = 70
```

Comment:
There's generally lots of fluctuation in run times and the most recent was 65 minutes.  I'd bump this at least a bit more.  Maybe even 90.

```suggestion
job_timeout = 80
```